pytest_wdl/url_schemes/dx.py

```diff
@@ -19,18 +19,24 @@
 
 from typing import Sequence, Optional
 
-import dxpy
-
 from pytest_wdl.url_schemes import Method, Request, Response, UrlHandler
 
 
 class DxResponse(Response):
     def __init__(self, file_id: str, project_id: Optional[str] = None):
         self.file_id = file_id
         self.project_id = project_id
+        # Import dxpy dynamically
+        self._dxpy = None
+
+    def dxpy(self):
+        if self._dxpy is None:
+            import importlib
+            self._dxpy = importlib.import_module("dxpy")
```

Comment:
Maybe add an informative error:
```suggestion
            try:
                self._dxpy = importlib.import_module("dxpy")
            except ModuleNotFoundError:
                raise ModuleNotFoundError(
                    "dx url scheme requires dxpy, which is not installed. "
                    "Update with pip install pytest-wdl[dx]")
```